package.json

```diff
@@ -3,6 +3,9 @@
   "version": "1.8.0",
   "description": "Shows Wikipedia article preview in a popup",
   "main": "dist/wikipedia-preview.production.js",
+  "engines": {
+    "node": "<17"
```

Comment:
Is this PR still relevant since this is also included in #177 ?

Reply:
this is still needed before the #177, as we tell the current version need older node version.

#177 has new node version setting